ui/perfherder/constants.js

```diff
@@ -0,0 +1,12 @@
+export const tValueCareMin = 3; // Anything below this is "low" in confidence
+export const tValueConfidence = 5; // Anything above this is "high" in confidence
+
+export const phTimeRanges = [
```

Comment:
I think it might be best to leave `phTimeRanges` in the existing constants file for now (given it's already not angular, which is the initial objective), and then move all the perf-related constants over at a later point once it's clearer which can be inlined in their component vs not?

Reply:
Fair enough :)